file-id/bin/file_id.rs

```diff
@@ -22,6 +22,6 @@ fn print_file_id(path: &str) {
 fn print_result(result: io::Result<FileId>) {
     match result {
         Ok(file_id) => println!("{file_id:?}"),
-        Err(error) => println!("Error: {error}"),
+        Err(error) => eprintln!("Error: {error}"),
```

Comment:
I mean, you can do that but this is just an example, so it won't matter at all. (Also there is an unrelated import change in that commit.)